examples/kubernetes/ceph-mds-v1-dp.yaml

```diff
@@ -0,0 +1,75 @@
+---
+kind: Deployment
+apiVersion: extensions/v1beta1
+metadata:
+  labels:
+    app: ceph
+    daemon: mds 
+  name: ceph-mds
+  namespace: ceph
+spec:
+  replicas: 1
+  template:
+    metadata:
+      name: ceph-mds
+      namespace: ceph
+      labels:
+        app: ceph
+        daemon: mds
+    spec:
+      nodeSelector:
+        node-type: storage
+      serviceAccount: default
+      volumes:
+        - name: ceph-conf
+          secret:
+            secretName: ceph-conf-combined
+        - name: ceph-bootstrap-osd-keyring
+          secret:
+            secretName: ceph-bootstrap-osd-keyring
+        - name: ceph-bootstrap-mds-keyring
+          secret:
+            secretName: ceph-bootstrap-mds-keyring
+        - name: ceph-bootstrap-rgw-keyring
+          secret:
+            secretName: ceph-bootstrap-rgw-keyring
+      containers:
+        - name: ceph-mon
+          image: quay.io/cornelius/ceph-daemon-test:latest
```

Comment:
Once testing is finished and a container is built by ceph-docker proper, we probably want a follow up PR to pin this to a particular tag.